server/src/test/java/de/uftos/services/TagServiceTests.java

```diff
@@ -0,0 +1,113 @@
+package de.uftos.services;
+
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import de.uftos.dto.requestdtos.TagRequestDto;
+import de.uftos.entities.Break;
+import de.uftos.entities.Server;
+import de.uftos.entities.Tag;
+import de.uftos.entities.TimetableMetadata;
+import de.uftos.repositories.database.ServerRepository;
+import de.uftos.repositories.database.TagRepository;
+import java.util.List;
+import java.util.Optional;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.ArgumentCaptor;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.mockito.junit.jupiter.MockitoSettings;
+import org.mockito.quality.Strictness;
+import org.springframework.web.server.ResponseStatusException;
+
+@SuppressWarnings("checkstyle:MissingJavadocType")
+@ExtendWith(MockitoExtension.class)
+@MockitoSettings(strictness = Strictness.LENIENT)
+public class TagServiceTests {
+  @Mock
+  private TagRepository tagRepository;
+
+  @Mock
+  private ServerRepository serverRepository;
+
+  @InjectMocks
+  private TagService tagService;
+
+  @BeforeEach
+  void setUp() {
+    Tag tag = new Tag("", "Sehbehinderung");
```

Comment:
Minor nit: I personally prefer having tests (basically everything non user-facing) be completely in English.